### PR TITLE
327/shadow filters

### PR DESCRIPTION
**Pull request by Alotor**
Adds shadow options for Rects, Circles, Frames, Paths

https://tree.taiga.io/project/uxboxproject/us/327?no-milestone=1

### CODE REVIEW

File: frontend/src/app/main/ui/workspace/colorpalette.cljs [hunk 1]
```diff
@@ -48,16 +48,16 @@
...
+        select-color
         (fn [event]
-          (if (kbd/shift? event)
-            (st/emit! (udw/update-color-on-selected-shapes {:stroke-color (:value color)
-                                                            :stroke-color-ref-file (:file-id color)
-                                                            :stroke-color-ref-id (:id color)}))
-            (st/emit! (udw/update-color-on-selected-shapes {:fill-color (:value color)
-                                                            :fill-color-ref-file (:file-id color)
-                                                            :fill-color-ref-id (:id color)}))))]
+          (println "item" id file-id)
```
Comment (niwinz) [inline frontend/src/app/main/ui/workspace/colorpalette.cljs]: :fire:

File: frontend/src/app/main/ui/workspace/shapes/path.cljs [hunk 2]
```diff
@@ -41,10 +42,13 @@
                              (do
                                (dom/stop-propagation event)
                                (dom/prevent-default event)
-                               (st/emit! (dw/start-edition-mode (:id shape)))))))]
+                               (st/emit! (dw/start-edition-mode (:id shape)))))))
+        filter-id (mf/use-var (filters/get-filter-id))]
```
Comment (niwinz) [inline frontend/src/app/main/ui/workspace/shapes/path.cljs]: Why `use-var` is used here?

File: frontend/src/app/main/ui/workspace/colorpalette.cljs (hunk 1 above)
Comment (Alotor) [reply frontend/src/app/main/ui/workspace/colorpalette.cljs]: OK

File: frontend/src/app/main/ui/workspace/shapes/path.cljs (hunk 2 above)
Comment (Alotor) [reply frontend/src/app/main/ui/workspace/shapes/path.cljs]: I'll change it to `use-memo`

**Pull request merged**